src/components/vpncard.js

```diff
@@ -0,0 +1,181 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import {
+  html,
+  css,
+  LitElement,
+  classMap,
+  styleMap,
+} from "../vendor/lit-all.min.js";
+
+/**
+ * @typedef {import("../background/vpncontroller/states.js").VPNState} VPNState
+ */
+
+export class VPNCard extends LitElement {
+  static properties = {
+    enabled: { type: Boolean },
+    connectedSince: { type: Date },
+  };
+
+  constructor() {
+    super();
+    this.enabled = false;
+    this.connectedSince = Date.now();
+  }
+  #intervalHandle = null;
+
+  updated(changedProperties) {
+    if (!changedProperties.has("enabled")) {
+      return;
+    }
+    if (this.enabled) {
```

Comment:
To show a timer we have 2 options: 
we have the `00:00:01` time as a property and update that property periodically. That will trigger `render()`. 
Oooor, in render we calculate `00:00:01` based on the elapsed time, that way we need to periodically request a re-render of the component since time elapses :)